src/server/server_family.h

```diff
@@ -173,7 +173,7 @@ class ServerFamily {
 
   void SyncGeneric(std::string_view repl_master_id, uint64_t offs, ConnectionContext* cntx);
 
-  std::error_code LoadRdb(const std::string& rdb_file);
+  std::error_code LoadRdb(const std::string& rdb_file, size_t* keys_loaded = nullptr);
```

Comment:
I always liked the rust'y/modern cpp style with `io::Result<size_t>` a little more than passing pointers, but it can be a little inconvenient at times without the proper utils around it. No need to change, just a suggestion